lib/utils/compareReactions.js

```diff
@@ -0,0 +1,21 @@
+let like;
+const { LIKE } = require('../../constants');
+/**
+ *
+ *
+ * @static
+ * @param {string} newReaction
+ * @param {boolean} existingReaction
+ * @returns {boolean} true or false
+ * @memberof commentsReactionModel
+ */
+const compareReactions = (newReaction, existingReaction) => {
```

Comment:
Why is this still here?

Reply:
This should have been deleted. I'll remove this now